ship.config.js

```diff
@@ -4,6 +4,13 @@ const path = require('path');
 
 module.exports = {
   mergeStrategy: { toSameBranch: ['master', 'next'] },
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
```

Comment:
```suggestion
    return releaseType === 'patch' && fix === 0;
```

note sure if that's better or worse tbh 😂

Reply:
We still need to return true when `releaseType` is 'minor' or 'major' 😅